configs/recognition/csn/ircsn_ig65m_pretrained_bnfrozen_r152_32x2x1_58e_kinetics400_rgb.py

```diff
@@ -21,7 +21,7 @@
         init_std=0.01))
 # model training and testing settings
 train_cfg = None
-test_cfg = dict(average_clips=None)
+test_cfg = dict(average_clips='score')
```

Comment:
'score' means logit?

Reply:
> The default value of `average_clips` should be "prob" rather than "score" for 3D model. "prob" will get higher performance for 3D. And for 2D recognizers, just set `None` for `average_clips` is OK.

Sorry that I made the mistake, the default value should be 'prob'. I will update this PR.

Reply:
Yes.